typedload/datadumper.py

```diff
@@ -201,3 +206,7 @@ def _dataclassdump(l, value):
                 del r[pyname]
                 r[dataname] = tmp
     return r
+
+
+def _ipaddressdump(l, value):
```

Comment:
uhm, you defined this function but then in the handlers wrote a lamda which does the same.

Reply:
Oops, removing it.